src/client/pfs/pfs.proto

```diff
@@ -483,6 +485,17 @@ message GetObjectsRequest {
   uint64 total_size = 4;
 }
 
+message GetBlocksRequest {
+  repeated BlockRef blockRefs = 1;
+  uint64 offset_bytes = 2;
+  // The number of bytes we intend to read.
```

Comment:
`size_bytes` is a hard cap on the number of bytes that will be returned right? Comment above seems a bit ambiguous if that's the case.

Reply:
Not sure I understand what you are saying here. The size field is for the requester to say how many bytes they want to have available to read.

Reply:
```suggestion
  // The number of bytes requested
```

Yeah, I'm just nitpicking the comment. Sorry I should just make a direct suggestion if I'm going to be so nitty. See above, I like that a little better.